src/Symfony/Component/WebLink/Link.php

```diff
@@ -180,6 +180,11 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
+    public function getAttribute(string $attribute): string|\Stringable|int|float|bool|array|null
+    {
+        return $this->attributes[$attribute] ?? null;
```

Comment:
`false` could be returned instead of `null`. It has the semantics of "attribute not set".
```suggestion
        return $this->attributes[$attribute] ?? false;
```

Reply:
I think `null` is a better representation of the absence of value.

Alternatively, we may avoid adding a method that is not part of the PSR-13 LinkInterface implemented by that class, to push consumer code to be written against the interoperable interface (i.e. using `getAttributes()['rel'] ?? null`, which let's _them_ be responsible of choosing the appropriate fallback for their case)

Reply:
Method removed.